pages/signup.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react';
+import { Button, Form } from 'react-bootstrap';
+import { useAuth } from '../context/AuthContext';
+
+const Signup = () => {
+  //  const { user, login } = useAuth();
+  const { signup } = useAuth();
+  //   console.log('User:' + user);
+  const [data, setData] = useState({
+    email: '',
+    password: '',
+  });
+
+  const handleSignup = async (e: any) => {
+    e.preventDefault();
+
+    try {
+      await signup(data.email, data.password);
+    } catch (err) {
+      //   console.log(err);
```

Comment:
We need to show errors to the user - such as this one
<img width="646" alt="Screen Shot 2022-06-06 at 4 15 45 PM" src="https://user-images.githubusercontent.com/21961090/172244458-773dea1f-6c82-486d-8739-fcaef39476eb.png">

Reply:
Fixed these errors, next pull request would be making a file decided for helper functions so we can just build on top that file for any more errors we might encounter